web_dialog_size/README.rst

```diff
@@ -1,67 +0,0 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: https://www.gnu.org/licenses/agpl
-   :alt: License: AGPL-3
-
-=============
-Expand Dialog
-=============
-
-A module that lets the user expand/restore the dialog box size through a button
-in the upper right corner (mimicking most windows managers).
-It does also add draggable support to the dialogs.
-
-Configuration
-=============
-
-By default, the module respects the caller's `dialog_size` option. If you want
-to override this and have all dialogs maximized by default, set the configuration
-parameter `web_dialog_size.default_maximize` to `1`.
-
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/162/11.0
-
-Bug Tracker
-===========
-
-Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/web/issues>`_. In case of trouble, please
-check there if your issue has already been reported. If you spotted it first,
-help us smash it by providing detailed and welcomed feedback.
-
-Credits
-=======
-
-Images
-------
-
-* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
-
-Contributors
-------------
-
-* Anthony Muschang <anthony.muschang@acsone.eu>
-* Stéphane Bidoul <stephane.bidoul@acsone.eu>
-* Pedro M. Baeza <pedro.baeza@tecnativa.com>
-* Holger Brunn <hbrunn@therp.nl>
-* Siddharth Bhalgami <siddharth.bhalgami@gmail.com>
-* Wolfgang Pichler <wpichler@callino.at>
-* David Vidal <david.vidal@tecnativa.com>
-* Quentin Theuret <quentin.theuret@amaris.com>
-
-Do not contact contributors directly about support or help with technical issues.
-
-Maintainer
-----------
-
-.. image:: https://odoo-community.org/logo.png
-   :alt: Odoo Community Association
-   :target: https://odoo-community.org
-
-This module is maintained by the OCA.
-
-OCA, or the Odoo Community Association, is a nonprofit organization whose
-mission is to support the collaborative development of Odoo features and
-promote its widespread use.
-
-To contribute to this module, please visit https://odoo-community.org.
```

Comment:
Please don't remove totally this file, as runbot will fail. You have 2 options:

* Put a line like the template one: `**This file is going to be generated by oca-gen-addon-readme.**`
* Run oca-gen-addon-readme locally for generating the README and not letting the bot do it at night.

Reply:
why? doesn't it create it later automatically?

Reply:
Yeah, but meanwhile you will have a red build